### DISCUSSION TITLE
No utility classes were detected in your source files. Content path configurate with vite aliases

### DISCUSSION THREAD

**Discussion by prksv** [tailwindlabs/tailwindcss#11015] [Help] [answered]
I'm using vite aliases and I'm getting a warning:
`warn - No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration.
warn - https://tailwindcss.com/docs/content-configuration
`

I've tried every way to enter paths to get rid of this warning, nothing worked. How do I enter paths in order to remove this warning?

![Безымянный](https://user-images.githubusercontent.com/94474856/232311770-66447190-78aa-4b84-bfd6-f88df88fe872.png)

**Comment by wongjn** [comment]
Consider checking whether:

- At least 1 valid Tailwind class has been used in the `index.html` file.
- At least 1 valid Tailwind class has been used in any `.js`, `.jsx`, `.ts` or `.tsx` files in the `src` directory.

If so, please consider providing a remotely-hosted git repository such that we may be able to troubleshoot further.

**Reply by wongjn** [reply, marked as answer]
> Im using tailwind in src/App.tsx 7 line. This works correctly, but the warning is still there.
> 
> https://github.com/prksv/wintura-frontend[^1]

Thank you for providing that. I have just cloned the project, ran `npm install` and `npm run dev` and I did not get any warnings.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/11015#discussioncomment-5628364

**Reply by prksv** [reply]
it's magic. after restarting the PC, the warning is gone. sorry for your wasted time!

**Comment by prksv** [comment]
Im using tailwind in src/App.tsx 7 line. This works correctly, but the warning is still there.

https://github.com/prksv/wintura-frontend

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/11015#discussioncomment-5628376 to keep the reply chain together.
